js/symbol/shaping.js

```diff
@@ -24,6 +24,8 @@ function Shaping(positionedGlyphs, text, top, bottom, left, right) {
     this.right = right;
 }
 
+var newLine = 0x0a;
```

Comment:
`\r` should be ignored in theory so `\r\n` should work the same way, but I could add a test to make sure the results are the same.